types/react-timeago.d.ts

```diff
@@ -0,0 +1,43 @@
+declare module 'react-timeago' {
```

Comment:
interesting, why did you have to add this type definition? doesn't it ship with the package?

Reply:
The types that are provided are not shipped with the package, no - they're provided separately in the DefinitelyTyped `@types` package, and appear quite out of date. The entire `react-timeago` package seems a bit old and abandoned, but I didn't want to make more changes than I had to to get the app working, so I just generated `.d.ts` types from the `.flow` file shipped with the package distribution.